docs/src/getting_started.md

```diff
@@ -0,0 +1,189 @@
+# Getting Started with JumpProcesses in Julia
+
+Jumps (or point) processes are stochastic processes with discrete state changes
+driven by a `rate` function. Historically, jump processes have been developed in
+the context of dynamical systems to describe dynamics with sudden changes —
+the jumps — in a system's value at random times. In contrast, the development
+of point processes has been more focused on describing the occurrence of random
+events — the points — over a support. In reality, jump and point processes
+share many things in common which make JumpProcesses ideal for both.
+
+Processes involving multiple jumps are known as compound jump (or point)
+processes.
+
+```math
+du = \sum_{j} h_j(u,p,t) dN_j(t)
+```
+
+where ``N_j`` is a jump process with rate ``\lambda_i(u,p,t)``.
+
+The homogeneous Poisson process is the canonical point process with a constant
+rate of change. A compound Poisson process is a continuous-time Markov Chain
+where the time to the next jump is exponentially distributed as determined by
+the aggregate rate. In the statistics literature, the composition of Poisson
+processes is described by the superposition theorem. Simulation algorithms for
+these types of processes are known in biology and chemistry as Gillespie methods
+or Stochastic Simulation Algorithms (SSA), with the time evolution that the
+probability these processes are in a given state at a given time satisfying the
+Chemical Master Equation (CME).
```

Comment:
This is likely not going to make much sense to anyone who doesn't know point-processes, which would be many of our users.